odo/backends/tests/test_dask.py

```diff
@@ -2,15 +2,19 @@
 
 import pytest
 pytest.importorskip('dask')
+pytest.importorskip('dask.bag')
```

Comment:
Yes, like that.

At the moment `dask` does not list all of the dependencies of its subprojects.  If one `pip` or `conda` installs dask you don't necessarily get all of `dill`, `numpy`, `bcolz` required for each of `dask.bag`, `dask.array`, `dask.frame` respectively.  Importing dask doesn't necessarily import all of the subprojects though.  Users will get the `ImportError` only if they try to import a subproject for which their environment does not contain the necessary dependencies.